### DISCUSSION TITLE
RTMP stream source working for enterprise 2.5.3 but not 2.6.2

### DISCUSSION THREAD

**Discussion by riftdea-isj** [ant-media/Ant-Media-Server#5537] [Q&A] [answered]
Using this source link below can play stream for 2.5.3, but offline forever on 2.6.2.
rtmp://IPsource:18935/live/stream0

Would like to know if any changes in recent update, I've missed the in-between versions. Would need it the stream to work on version 2.6.2.
Thank you.

**Comment by Mohit-3196** [comment]
Hi @riftdea-isj ,
How are you?

Please check this discussion https://github.com/orgs/ant-media/discussions/4874#discussioncomment-5230160
If it doesn't help, please let me know.

Thank you.

**Reply by riftdea-isj** [reply]
Changing to .rtmpPlaybackEnabled=true does not help, now even my rtps://ip/stream is not able to be play anymore.

**Reply by riftdea-isj** [reply]
Hi @Mohit-3196 
I'm good thank you.

I've tested with the link you gave me, its not working still. Would need you to look at it further.

Thank you.

**Reply by yashtandon113** [reply]
@riftdea-isj 
If you are trying to pull the RTMP stream on a server then yes, their is an issue in v2.6.2 but it was working fine in v2.5.3.
We have investigated and fixed the related GitHub issue https://github.com/ant-media/Ant-Media-Server/issues/5500.

It will be reviewed and merged soon. You can check latest snapshot change logs [here](https://antmedia.io/changelog/changelog.html).

Once it will be merged, you can download the latest snapshot from your antmedia.io account.

**Reply by riftdea-isj** [reply]
Hi @yashtandon113 
I've just tried with v2.6.4 enterprise. It still not working for RTMP.

**Reply by yashtandon113** [reply]
@riftdea-isj 
Yes, you are right. As I mentioned in my answer, the fix has been implemented but the PR is not merged yet. It is under review and will be merged soon.

**Reply by Mohit-3196** [reply]
Thank you @yashtandon113

**Reply by riftdea-isj** [reply]
Hi @yashtandon113 
Ok, great do let me know when it successfully solved and when new version come out. I'll make as answer by then.
Thank you very much.

**Reply by riftdea-isj** [reply]
HI @yashtandon113 
Is it possible to get a script for ./antmedia_install that allow me to install v2.5.3 enterprise? Cause I try installed old enterprise version with the latest script, will automatically force me to install community version of antmedia. I'm kind of in desperate need of installing v2.5.3, due to RTMP. 

We need RTMP for our streaming and need to host the antmedia in cloud, but now can't cause of this restriction. Need your help on this, Thank you very much.

**Reply by Mohit-3196** [reply]
Hi @riftdea-isj ,
You will have to manually pass the v2.5.3 zip file with the installation script in order to install 2.5.3
`sudo ./install_ant-media-server.sh -i <ANT_MEDIA_SERVER_ZIP_FILE>`

Please share your email address with me and I will share the installation zip file of v2.5.3.

Alternatively, you can also restore to the previous version using the backup file as instructed here https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/#how-do-i-restore-ant-media-server-if-needed

Let me know if you need any help.

Thank you

**Reply by riftdea-isj** [reply]
@Mohit-3196 
Sorry for the late reply, this is my email: Rift.dea@isj-technology.com 
Thank you very much.

**Reply by Mohit-3196** [reply, marked as answer]
No worries @riftdea-isj ,
I have mailed you the file along with the instructions.

Thank you

**Reply by riftdea-isj** [reply]
@Mohit-3196 
Thank you very much for your help!!
